src/internal/init.c

```diff
@@ -195,6 +195,7 @@ static const R_CallMethodDef r_callables[] = {
   {"ffi_set_names",                    (DL_FUNC) &ffi_set_names, 4},
   {"ffi_sexp_iterate",                 (DL_FUNC) &ffi_sexp_iterate, 2},
   {"ffi_squash",                       (DL_FUNC) &ffi_squash, 4},
+  {"ffi_standalone_check_number_1.0.7", (DL_FUNC) &ffi_standalone_check_number, 7},
```

Comment:
Do you care about the indentation of `(DL_FUNC)` being off?

Reply:
yes but I didn't feel like reindenting everything in this PR